<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Supper_dialog</class>
 <widget class="QWidget" name="Supper_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>255</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>384</width>
    <height>255</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>384</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление продукта на ужин</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #d6ffc5</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>77</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="dialog_layout">
    <item>
     <widget class="QPushButton" name="yes_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Да</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="no_btn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Нет</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Указанного продукта(блюда) нету в&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;базе данных и/или он неверно указан&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="dialog_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Желайте ли вы добавить его в базу данных?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>340</width>
     <height>98</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="Supper_layout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Добавить продукт(блюдо) на ужин:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="product">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Добавить массу(в граммах) продукта(блюда):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_product">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Указанный продукт уже занесён&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
